docs/how-to-guides/wsl-integration/use-wsl-profiles.md

```diff
@@ -38,11 +26,14 @@ Then complete the relevant fields:
 - **rootfs image**: The rootfs image to be installed. These correspond to the available WSL Ubuntu releases on the Microsoft Store. The Ubuntu image corresponds to the latest release available on the store and will give the WSL instance the ability to upgrade to the next release when it's available. If you have a custom rootfs image, you can provide the URL instead by selecting **From URL**. If you want multiple copies of the same rootfs image, you must use the **From URL** selection and provide a unique name.
   - **Image name**: The name of the rootfs image that will be installed on the WSL instance.
   - **rootfs image URL**: The URL of the rootfs image that will be installed on the WSL instance. This URL must be reachable by the Windows host machine. Organizations can also host their own custom rootfs images and reference them here. If your environment uses a WSL `manifest.json` file to define which distributions are available for `wsl --install`, you can point Windows to your own manifest URL. This is optional and independent of Landscape, but it allows administrators to present a controlled list of distributions to end users. For example (using a beta manifest URL):
+
     ```powershell
     $distributionListUrl = "https://raw.githubusercontent.com/wiki/canonical/ubuntu-pro-for-wsl/beta/manifest.json"
     Set-ItemProperty -Path "HKLM:SOFTWARE\Microsoft\Windows\CurrentVersion\Lxss" -Name DistributionListUrl -Value "$distributionListUrl" -Type String -Force
     ```
+
 - **cloud-init** (optional): The contents of the cloud-init to be supplied to the WSL instance. This can be uploaded as a file or inputted as text. If you don't include a cloud-init file, the rootfs image will be applied without any additional configuration.
+- **Compliance settings** : Whether or not the associated Windows instances should only have WSL instances that were created through Landscape.
```

Comment:
Ahh - I see this here. Going to go edit my comments on below (in case you read this first, I was asking about the concept of "strictness", which is new)